tests/sentry/integrations/slack/test_unfurl.py

```diff
@@ -165,10 +170,11 @@ def test_unfurl_discover_short_url(self, mock_generate_chart):
         ]
 
         with self.feature(
-            {
-                "organizations:discover": True,
-                "organizations:chart-unfurls": True,
-            }
+            [
+                "organizations:discover",
+                "organizations:discover-basic",
```

Comment:
without `discover-basic` this was actually testing the wrong version of the stats endpoint